### PR TITLE
ci: faster process_replay

### PR DESCRIPTION
**Pull request by maxime-desroches**

### CODE REVIEW

File: selfdrive/controls/controlsd.py [hunk 1]
```diff
@@ -204,12 +204,10 @@ def publish(self, CC, lac_log):
     self.pm.send('carControl', cc_send)
 
   def run(self):
-    rk = Ratekeeper(100, print_delay_threshold=None)
     while True:
       self.update()
       CC, lac_log = self.state_control()
       self.publish(CC, lac_log)
-      rk.keep_time()
```
Comment (sshane) [inline selfdrive/controls/controlsd.py:212]: should be `rk.monitor_time()`

**Review by sshane — CHANGES_REQUESTED** [review]

File: .github/workflows/selfdrive_tests.yaml [hunk 2]
```diff
@@ -210,9 +210,9 @@ jobs:
       run: |
         ${{ env.RUN }} "scons -j$(nproc)"
     - name: Run replay
-      timeout-minutes: 30
+      timeout-minutes: 2
       run: |
-        ${{ env.RUN }} "coverage run selfdrive/test/process_replay/test_processes.py -j$(nproc) && \
+        ${{ env.RUN }} "FILEREADER_CACHE=1 coverage run selfdrive/test/process_replay/test_processes.py -j$(nproc) && \
```
Comment (adeebshihadeh) [inline .github/workflows/selfdrive_tests.yaml:215]: isn't this set globally?

Comment (sshane) [reply .github/workflows/selfdrive_tests.yaml:215]: yep in env.RUN. why isn't it the default though?

**Comment by sshane** [comment]
I don't like that we're removing coverage

**Comment by adeebshihadeh** [comment]
We only need all the cars for `card` and maybe a couple others with car-specific code paths.

**Comment by sshane** [comment]
controlsd is also pretty important since it deals with car-specific controllers, was useful in checking what changed in https://github.com/commaai/openpilot/pull/33970

**Comment by maxime-desroches** [comment]
I think it is ok to reduce coverage in CI while keeping the possibility for someone to run longer tests easily offline on their own (like `controlsd` on all segments) if they feel like a change might benefit from that type of coverage (a bit like how we currently skip all slow pytest tests during the `unit_tests` job)

**Pull request merged**